CONTRIBUTING_DOCS.md

```diff
@@ -132,27 +132,50 @@ Examples:
 - Calico API schema reference (policy, ip pool, etcd)
 
 
-## Doc site architecture
+## Organizational changes
 
-The naming and layout of these navbars are stored in `_data/$VERSION/navbars/*`. Jekyll automatically stores information from the `_data` directory in an accessible variable called `site.data`. The top-level layout (`_layout/docwithnav.html`) will iterate through all the files in `site.data[version].navbars` to construct the sidebar based on which version is being viewed.
+### Creating new pages
+
+- To create a top level splash page for a URL path, simply name the file `index.md`.
+
+- If you are adding a new page to the current release directory, add the following metadata just under the `title` of the page: `redirect_from: latest/<path-to-new-file>`. This ensures that readers who use `latest` in their paths instead of the version number of the current release get redirected to your new page. If you do not add this, you may experience `htmlproofer` errors.
+
+
+### Deleting and renaming pages
+
+If you need to delete or rename a directory or file:
+
+- Ensure that you [adjust the side navigation bar to match](#side-navigation-bar).
+
+- If the page exists in previous versions, add the following metadata just under the `title` of the files in the directories of the previous versions: `no-canonical: true`. Otherwise, `htmlproofer` will fail.
+
+### Side navigation bar
```

Comment:
I'm curious if "Top navigation bar" is more accurate ...

Reply:
I think it is the side navigation bar that we're talking about here. As I recall, the top navigation bar is manipulated in a different way that is somewhat complex. I have done it, but I immediately forgot how I did it. :) It's much more likely that people will need to manipulate the side navigation versus the top.